examples/config.json

```
@@ -78,7 +78,9 @@
         "misc_orthographic_camera": "Orthographic camera",
         "misc_custom_controls": "Define custom controls",
         "misc_custom_label": "Custom label popup",
-        "misc_camera_traveling": "Camera traveling"
+        "misc_camera_traveling": "Camera traveling",
+        "underground_25d_map": "Underground visualization"
```

```suggestion
        "misc_underground_25d_map": "Underground visualization"
```

Could you please change the example filename to respect the naming convention of the miscellaneous examples?